tests/unit/onboardingApiTest.ts

```diff
@@ -0,0 +1,61 @@
+/* eslint-disable @typescript-eslint/naming-convention */
+import Onyx from 'react-native-onyx';
+import type {OnyxEntry} from 'react-native-onyx';
+import CONST from '@src/CONST';
+import * as Report from '@src/libs/actions/Report';
+import ONYXKEYS from '@src/ONYXKEYS';
+import type {ReportActions} from '@src/types/onyx/ReportAction';
+import * as TestHelper from '../utils/TestHelper';
+import waitForBatchedUpdates from '../utils/waitForBatchedUpdates';
+
+const ESH_EMAIL = 'eshgupta1217@gmail.com';
+const ESH_ACCOUNT_ID = 1;
```

Comment:
@Krishna2323 Either you've used your own details here instead of another contributor's, or you can use a placeholder like `TEST_USER_ACCOUNT_ID`.

Reply:
Instead of creating a new test file named `onboardingApiTest`, what do you think about adding the test directly under `tests/actions/ReportTest.ts`, since we're only testing `completeOnboarding` as part of the Report actions?